doc/src/tutorial/index.rst

```diff
@@ -17,3 +17,4 @@
    solvers.rst
    matrices.rst
    manipulation.rst
+   numeric-computation.rst
```

Comment:
This isn't supposed to go in the tutorial. The tutorial is an introductory tutorial. The numeric computation page is a guide (see https://documentation.divio.com/). I agree it doesn't blog in `modules`, which is the reference, but it doesn't belong here either. We should have a separate sections for how-to guides.

Reply:
What about `special-topics`, then?

Reply:
Then what about putting it into `special_topics`?

Reply:
That's also a good name. We just need some sort of top level directory for user guides so that people aren't tempted to put them in the tutorial (it happens too often).

Reply:
I renamed it to `how-to guides` and moved `numeric-computation.rst` to there.